package.nls.json

```
@@ -101,6 +102,7 @@
     "AWS.ecs.enableEcsExec": "Enable Command Execution",
     "AWS.ecs.disableEcsExec": "Disable Command Execution",
     "AWS.ecs.runCommandInContainer": "Run command in container",
+    "AWS.ecs.openTaskInTerminal": "Open interactive session in terminal",
```

- can we just name this "Open Terminal"? That also somewhat matches vscode's own wording.
- this is an action, right? So I guess it needs to be Title Case (same for the one above it)


```suggestion
    "AWS.ecs.openTaskInTerminal": "Open Terminal",
```